packages/plugin-ext/src/common/plugin-protocol.ts

```diff
@@ -185,7 +185,7 @@ export interface PluginPackageLanguageContributionConfiguration {
 export interface PluginTaskDefinitionContribution {
     type: string;
     required: string[];
-    properties: {
+    properties?: {
```

Comment:
@elaihau maybe you can comment as to why the `properties` was initially mandatory?

Reply:
I would, but I have no clue why. I just noticed that ESLint, the most popular extension does not provide these properties.

Reply:
> > > Hi @JonasHelming, thank you for your contribution :)
> > > Can you please be sure to fill out the PR template, the fields for `What it does` and `How to test` are empty. It helps during the review process.
> > > In addition, there are typos present in your commit message: `Return an empty array if undefined`
> > 
> > 
> > Hmm, where is the typo?
> 
> `undefinied ` should be `undefined`.

fixed, sorry :-)

Reply:
> I would, but I have no clue why. I just noticed that ESLint, the most popular extension does not provide these properties.

No problem, just wanted to ping the original author of the code to determine if it was mandatory for a reason. I don't see the harm in making it optional :)